src/trythis.ts

```diff
@@ -1,5 +1,5 @@
 // Match everything after "Try this" until the next unindented line
-export const magicWord = 'Try this: ';
+export const magicWord = 'Try this:';
```

Comment:
Removing this space is necessary for the suggestions in https://github.com/leanprover-community/mathlib/pull/4093 to trigger.